config/routes.rb

```diff
@@ -308,7 +308,7 @@
 
   get 'post' => 'editor#post', :as => :editor_post
   post 'post' => 'editor#post', :as => :editor_path
-  get 'post/simple' => 'editor#simple'
+  get '/simple_post' => 'editor#simple'
```

Comment:
Hmm, what was the problem here? I liked `/post/simple` -- was it not finding this in the correct precedence order? Would it work if you moved /post/simple above line 309?

Reply:
The capybara test was failing with a error undefined route '/post/undefined` trying to find another fix

Reply:
The error is ActionController::RoutingError:         ActionController::RoutingError: No route matches [GET] "/post/undefined"

Reply:
Weird! Let me see your capybara tests... can you mark what line of tests fails?